Mimecast/mimecast-email-security/ingest/parser.yml

```
@@ -115,3 +132,23 @@ stages:
       - set:
           url.original: "{{ parse_rejectioninfo.message.Url }}"
         filter: '{{ "Url" in parse_event.message.rejectionInfo}}'
+
+      - set:
+          mimecast.siem.spam_processing_detail: "{{parse_spam_processing_detail.details}}"
+        filter: '{{ "spamProcessingDetail" in parse_event.message and parse_event.message.spamProcessingDetail != "" and not parse_event.message.spamProcessingDetail.startswith("\\\"") }}'
+
+      - set:
+          source.user.domain: "{{parse_event.message.senderEnvelope.split('@')[-1].split('.')[0]}}"
```

You should keep every level of the domain name.
```suggestion
          source.user.domain: "{{parse_event.message.senderEnvelope.split('@')[-1]}}"
```